integration-tests/README.md

```diff
@@ -0,0 +1,295 @@
+# Integration Tests for Operator
+
+This documentation describes the functional use cases that are covered in integration testing for Weblogic Operator. The tests are written in java(Junit tests) and driven by maven profile. All the use cases covered in run.sh will be covered in java integration tests. Currently, QUICKTEST use cases and some of non-quick test(till domain4) use cases are covered in java tests.
+
+# Environments
+
+The tests currently runs in three modes, "Wercker", "Jenkins", and "standalone" Oracle Linux, where the mode is controlled by the WERCKER and JENKINS environment variables described below. The default is "standalone".
+
+* "Standalone" Oracle Linux, i.e, run the tests manually with mvn command. 
+* Wercker - https://app.wercker.com/Oracle/weblogic-kubernetes-operator/runs - integration-test-java is the pipeline name
+* Jenkins - http://wls-jenkins.us.oracle.com/view/weblogic-operator/job/weblogic-kubernetes-operator-javatest/
+
+# Use Cases
+
+Currently java integration tests cover all QUICKTEST and some non-QUICKTEST use cases from run.sh. The below are the use cases:
+
+* create operator operator1 which manages default and test1 namespaces, verify its deployed successfully, pod created, operator Ready and verify external REST service if configured
+* create domain domain1 in default namespace and verify the pods, services are created and servers are in Ready
+* verify admin external service by accessing admin REST endpoint with nodeport in URL
+* verify admin t3 channel port by exec into the admin pod and deploying webapp using the channel port for WLST
+* verify web app load balancing  
+* verify domain life cycle(destroy and create) should not any impact on Operator managing the domain and web app load balancing and admin external service
+* cluster scale up/down using Operator REST endpoint, webapp load balancing should adjust accordingly. (run.sh does scaling by editing the replicas in domain-custom-resource.yaml.)
+* Operator life cycle(destroy and create) should not impact the running domain
+
+Non-quick test use cases
```

Comment:
Non-quick?  :). Can we call if "Full test"?

Reply:
Changed to Full test